neon_homeassistant_skill/__init__.py

```diff
@@ -491,9 +477,13 @@ def handle_assist_intent(self, message):  # noqa
     #         self.speak_dialog("device.not.found", data={"device": device})
 
     def _handle_assist_error(self, _):
-        self.speak(
-            "Home Assistant returned an error. Please check the enclosure or PHAL logs for more information."
-        )
+        self.speak("Home Assistant returned an error. Please check the enclosure or PHAL logs for more information.")
 
     def _get_ha_value_from_percentage_brightness(self, brightness):
         return round(int(brightness)) / 100 * 255
+
+
+if __name__ == "__main__":
+    from ovos_utils.messagebus import FakeBus
+
+    NeonHomeAssistantSkill(bus=FakeBus(), skill_id="neon_homeassistant_skill.test")
```

Comment:
Leftover test code?

Reply:
Yes, but I intended to keep it because it doesn't harm anything